### PR TITLE
edge-19.6.4

### PR DESCRIPTION
**Pull request by adleong**
## edge-19.6.4

This release adds support for the SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md) API.  Creating a TrafficSplit resource will cause Linkerd to
split traffic between the specified backend services.  Please see [the spec](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
for more details.

* CLI
  * Added a check to install to prevent installing multiple control planes into
    different namespaces
  * Added support for passing a URL directly to linkerd inject
  * Added a `--all-namespaces` flag to `linkerd edges`
* Controller
  * Added support for the SMI TrafficSplit API which allows users to define
    traffic splits in TrafficSplit custom resources
* Proxy
* Web UI
  * Improved UI for Edges table in dashboard by changing column names, adding a
    "Secured" icon and showing an empty Edges table in the case of no returned
    edges

Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+## edge-19.6.4
+
+This release adds support for the SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+API.  Creating a TrafficSplit resource will cause Linkerd to split traffic
+between the specified backend services.  Please see [the spec](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+for more details.
+
+* CLI
+  * Added a check to install to prevent installing multiple control planes into
```
Comment (scottcarol) [inline CHANGES.md]: I would backtick `install`

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+    different namespaces
+  * Added support for passing a URL directly to linkerd inject
```
Comment (scottcarol) [inline CHANGES.md]: `linkerd inject`

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+  * Added a `--all-namespaces` flag to `linkerd edges`
```
Comment (scottcarol) [inline CHANGES.md]: The tiniest nit but... I think it should be `an` and not `a`

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+* Controller
+  * Added support for the SMI TrafficSplit API which allows users to define
+    traffic splits in TrafficSplit custom resources
+* Proxy
```
Comment (scottcarol) [inline CHANGES.md]: this is blank?

Comment (admc) [inline CHANGES.md]: Do we normally areas like "Proxy" when there are no updates? Should we add something about traffic split support in the proxy?

File: CHANGES.md (hunk 2 above)
Comment (ihcsim) [inline CHANGES.md]: Shout out to Tarun (at Pothulapati).

File: CHANGES.md (hunk 4 above)
Comment (adleong) [reply CHANGES.md]: good catch.  traffic split spans controller and proxy but I decided to put it in the controller section.  I don't think we need it in both.  I'll remove the empty proxy bullet.

File: CHANGES.md (hunk 3 above)
Comment (kleimkuhler) [reply CHANGES.md]: Still tiny, but it seems like we have usually referred to new flags with `the`!

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,24 @@
+## edge-19.6.4
+
+This release adds support for the SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+API.  Creating a TrafficSplit resource will cause Linkerd to split traffic
+between the specified backend services.  Please see [the spec](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
```
Comment (kleimkuhler) [inline CHANGES.md]: Looks like extra space before "Please ..."

**Pull request merged**